cfn-resources/cloud-backup-schedule/test/cfn-test-delete-inputs.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# cfn-test-delete-inputs.sh
+#
+# This tool deletes the mongodb resources used for `cfn test` as inputs.
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function usage {
+    echo "usage:$0 "
+}
+
+
+projectId=$(jq -r '.ProjectId' ./inputs/inputs_1_create.json)
+clusterName=$(jq -r '.ClusterName' ./inputs/inputs_1_create.json)
+
+echo $projectId
+echo $clusterName
+
+#delete Cluster
+if atlas clusters delete "$clusterName" --projectId "${projectId}" --force
+then
+    echo "$clusterName cluster deletion OK"
+else
+    (echo "Failed cleaning cluster:$clusterName" && exit 1)
+fi
+
+echo "Waiting for cluster to get deleted"
+sleep 900
```

Comment:
900 is like 15 mintues? it takes this long?

Reply:
yes some times